addons/website/controllers/main.py

```diff
@@ -227,6 +230,10 @@ def sitemap_xml_index(self, **kwargs):
         View = request.env['ir.ui.view'].sudo()
         mimetype = 'application/xml;charset=utf-8'
         content = None
+        url_root = request.httprequest.url_root
+        # cache per domain
```

Comment:
Maybe an uppercase later :innocent: :flushed:

Reply:
Rephrased it a bit too so it's more clear